### ISSUE TITLE
Scrolling up while composing a comment can remove your partially-composed comment

### ISSUE THREAD

**Issue by andrewmoise** [Xyphyn/photon#457]
**Describe the bug**

Scrolling up while composing a comment can remove your partially-composed comment

**To Reproduce**

* Reply to a comment within a long comments section
* Scroll back up to the top of the page
* Scroll back down to your draft comment
* Your comment is gone

**Expected behavior**

Your comment wouldn't be gone

**Versions**
- Browser: Chrome
- Photon version: 1.31.4

**Comment by Xyphyn** [comment]
Good idea to fix in #729 along with reactive stores (talking to myself)
